scripts/fast-reboot

```diff
@@ -391,6 +412,11 @@ init_warm_reboot_states
 setup_control_plane_assistant
 
 if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+    debug "Disabling swss auto restart ..."
+    docker exec swss supervisorctl stop supervisor-proc-exit-listener &> /dev/null || /bin/true
+    sed -i -e "s/\<RestartSec=30\>/RestartSec=300/" /etc/systemd/system/swss.service || /bin/true
+    systemctl daemon-reload || /bin/true
```

Comment:
Doing it here is too early. We can defer it to after set +e before shutting down swss.

Reply:
Moved this to after setting `+e`